.gitignore

```diff
@@ -2,3 +2,4 @@
 TODO
 /prometheus-meta-operator
 !vendor/**
+.idea/
```

Comment:
I think this should be in your personal global gitignore list.
See https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/ for more info.

Reply:
Thanks